spec/features/write_to_councillors_spec.rb

```diff
@@ -16,13 +16,16 @@
     end
 
     scenario "can’t see councillor messages sections" do
-      page.should_not have_content("Write to your elected councillors about this application")
+      page.should_not have_content("Who should this go to?")
       # TODO: and you should not be able to write and submit a message.
     end
   end
 
   context "when logged in as admin" do
     background do
+      @council_or_councillor_explanation = "Write to the council
```

Comment:
I originally I did it this way cause I was writing it in the test before the view and I found i easier to read/write that way.

But looking at it now I agree its confusing @henare . I think inlining the string is the simplest thing to do.